CHANGELOG.md

```diff
@@ -45,6 +46,47 @@ RELEASE CHANGELOG TEMPLATE:
 ### Documentation
 -->
 
+<a name="v2.3.0"></a>
+# [2.3.0](https://github.com/ligato/vpp-agent/compare/v2.2.0...v2.3.0) (2019-10-04)
+
+### Compatibility
+- **VPP 20.01** (rc0)
+- **VPP 19.08** (default)
```

Comment:
We are not compatible with `19.08` anymore, this should be `19.08.1` as it contains a breaking change.